sources/osgShadow/ShadowMapAtlas.js

```diff
@@ -409,6 +409,13 @@ utils.createPrototypeObject(
                     idx = this._lights.indexOf(light);
                     if (idx !== -1) this._lights.splice(idx, 1);
 
+                    if (this._viewportDimension.length > idx) {
+                        this._viewportDimension.splice(idx, 1);
```

Comment:
why not just keeping a camera array ? inside you can have viewport and scissor

Reply:
it's vec4 I store and keep. 
(it's the data of the uniform "mapSize" for shadow receiving (clamping on tiles in atlas))